doc/api/process.md

```diff
@@ -36,6 +36,26 @@ termination, such as calling [`process.exit()`][] or uncaught exceptions.
 The `'beforeExit'` should *not* be used as an alternative to the `'exit'` event
 unless the intention is to schedule additional work.
 
+```js
+process.on('beforeExit', (code) => {
+  console.log('Process beforeExit event with code: ', code);
+});
+
+(function doSomething() {
+  console.log('This message is displayed first.');
+})();
+
+// Prints:
+// doSomething: This message is displayed first.
```

Comment:
Nits: it seems `doSomething: ` and `beforeExit: ` parts are not actually printed?

Reply:
No, those parts are currently being printed. The problem is with the content of the commit message.  
Can you help me solve that?

Reply:
This is what I get on windows:
```
>cat test.js
﻿process.on('beforeExit', (code) => {
  console.log('Process beforeExit event with code: ', code);
});
(function doSomething() {
  console.log('This message is displayed first.');
})();

>node test.js
This message is displayed first.
Process beforeExit event with code:  0

>
```
About commit message see [Commit message guidelines](https://github.com/nodejs/node/blob/master/doc/guides/contributing/pull-requests.md#commit-message-guidelines). It seems `doc: add example for beforeExit event` would be OK.

Reply:
Well, thanks for your help, but if everything's okay, how can I get Travis CI accept my PR?

Reply:
One of Node.js collaborators will run CI on the PR)

Reply:
Ok thanks!!